src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    void drawFigure();
```

Comment:
The method name `drawFigure` could be simplified to `draw` since it's already in the context of an interface named `Drawable`, which implies that the action is related to drawing.